CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project("kafka-to-nexus")
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${PROJECT_SOURCE_DIR}/cmake")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror")
```

Comment:
In view of [Get your hands off CMAKE_CXX_FLAGS](https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#get-your-hands-off-cmake_cxx_flags) is there something we can do here?

Reply:
As far as I know, and as far as I've been able to google, CMake doesn't offer any abstraction for warning/error levels, so I think this is all we can do. Thanks for prompting me to check though.